app/helpers/katello/concerns/smart_proxy_helper_extensions.rb

```diff
@@ -0,0 +1,28 @@
+module Katello
+  module Concerns
+    module SmartProxyHelperExtensions
+      def disks(storage)
+        mount_points = {}
+        mount_points.default = []
+        disks = []
+        storage.each do |name, values|
+          mount = values['mounted']
+          next if mount_points.keys.include?(mount)
+          mount_points[mount] << name
+          values['header'] = "#{mount_points[mount].to_sentence} (#{values['path']} on #{values['filesystem']})"
+          values['total'] = values.delete('1g-blocks')
+          disks << values
```

Comment:
I do see in the UI:

pulp_dir, pulp_content_dir, and mongodb_dir (/var/lib/mongodb on /dev/vda3)

but i might expect the list of directories rather than their config values.